src/docs/sphinx/dev_guide/misc_tasks.rst

```diff
@@ -12,19 +12,6 @@ Miscellaneous Development Items
 This section describes various development tasks that need to be 
 performed at times and which are not covered in other sections.
 
-We need to check that answers to the following questions are contained in
-this section and are clear:
-
-  * How does one add a new compiler or platform to the mix?
-  * How does one build a new set of TPLs for a single platform or compiler
-    for testing?
-  * What is the procedure for changing versions of one or more TPLs?
-  * How do we keep things straight when using different TPL versions for 
-    different branches?
-  * How to use the scripts for team TPL support vs. local development 
-    and experimentation?
-  * Others?
-
 
 ===================
```

Comment:
Does this line have an errorneous space as the first character? It appears to render fine in RTD.

Reply:
It does not. I think it just looks funny in this rendering.